js/app.js

```diff
@@ -114,71 +114,76 @@ cardItem.forEach((item, j) => {
   callCardBody(j);
```

Comment:
- [ ] For best practices, please consider removing the comment in ```line 65``` if it's not used, comments that are not necessary make your code messy